utils/forest_shim/src/sector.rs

```diff
@@ -0,0 +1,102 @@
+// Copyright 2019-2023 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+pub use fvm_shared::sector::RegisteredSealProof as RegisteredSealProofV2;
+pub use fvm_shared::sector::SectorInfo as SectorInfoV2;
+pub use fvm_shared3::sector::RegisteredSealProof as RegisteredSealProofV3;
+pub use fvm_shared3::sector::SectorInfo as SectorInfoV3;
+use fvm_shared3::sector::SectorNumber;
+use std::ops::Deref;
+
+/// Represents a shim over RegisteredSealProof from fvm_shared with convenience methods to convert to an older version of the type
```

Comment:
It would be fantastic if you could add some tests or compiled documentation like I tried here https://github.com/ChainSafe/forest/pull/2473/files#diff-818bddc7286e55c350466d2b08b8a107c86e10fc0f85e3b117e4865dcc5ff865R27-R46

Not a big deal, though, given that those matches are exhaustive.

Reply:
Added

Reply:
rock-solid